database/docker-compose.yml

```diff
@@ -1,15 +1,14 @@
 services:
-  local_database:
+  database:
     image: postgres:latest
-    container_name: local_postgres
+    container_name: database
```

Comment:
I was a bit confused here. Are we using `database` now or still `local_postgres`? Let's commit on `database`. Then we would need to change the `.env.local` file again. :)

Reply:
Yeah ok, lets move with database.